edb/schema/types.py

```diff
@@ -315,6 +318,10 @@ def contains_array_of_tuples(self, schema: s_schema.Schema) -> bool:
         return self.contains_predicate(
             lambda x: x.is_array_of_tuples(schema), schema)
 
+    def contains_stdlib_type(self, schema: s_schema.Schema) -> bool:
+        return self.contains_predicate(
+            lambda x: x.is_stdlib_type(schema), schema)
```

Comment:
Where is this used?

Reply:
Hmm. I originally added this thinking I would need it in case people tried to get clever wth various expressions in UPDATE or DELETE statements, but I now believe that creativity to be lillegal. So, I agree. I think it's dead. I'll remove it.